app/src/main/java/agersant/polaris/navigation/NavigationExtensions.kt

```diff
@@ -0,0 +1,251 @@
+/*
```

Comment:
Can you add a comment near the top describing where exactly this code came from?

Reply:
Done